src/constants/chains.test.ts

```diff
@@ -0,0 +1,29 @@
+import { ChainId } from '@uniswap/sdk-core'
+
+import { getChainPriority } from './chains'
+
+// Define an array of test cases with chainId and expected priority
+const chainPriorityTestCases: [ChainId, number][] = [
+  [ChainId.MAINNET, 0],
+  [ChainId.GOERLI, 0],
+  [ChainId.SEPOLIA, 0],
+  [ChainId.POLYGON, 1],
+  [ChainId.POLYGON_MUMBAI, 1],
+  [ChainId.ARBITRUM_ONE, 2],
+  [ChainId.ARBITRUM_GOERLI, 2],
+  [ChainId.OPTIMISM, 3],
+  [ChainId.OPTIMISM_GOERLI, 3],
+  [ChainId.BNB, 4],
+  [ChainId.AVALANCHE, 5],
+  [ChainId.CELO, 6],
+  [ChainId.CELO_ALFAJORES, 6],
+]
+
+// Run the tests using test.each
+test.each(chainPriorityTestCases)(
+  'getChainPriority returns expected priority for a given ChainId',
```

Comment:
You should include the chain under test in the test name so failures are more easily fixed